src/webfinger/handler.test.ts

```diff
@@ -115,6 +116,90 @@ test("handleWebFinger()", async (t) => {
     assertEquals(onNotFoundCalled, null);
   });
 
+  const expectedForLocalhostWithPort = {
+    subject: "acct:someone@localhost:8000",
+    aliases: ["https://localhost:8000/users/someone"],
+    links: [
+      {
+        href: "https://localhost:8000/users/someone",
+        rel: "self",
+        type: "application/activity+json",
+      },
+      {
+        href: "https://localhost:8000/@someone",
+        rel: "http://webfinger.net/rel/profile-page",
+      },
+      {
+        href: "https://example.org/@someone",
+        rel: "alternate",
+        type: "text/html",
+      },
+      {
+        href: "https://localhost:8000/icon.jpg",
+        rel: "http://webfinger.net/rel/avatar",
+        type: "image/jpeg",
+      },
+    ],
+  };
+
+  await t.step("on localhost with port, ok: resource=acct:...", async () => {
+    const u = new URL("https://localhost:8000/.well-known/webfinger");
+    u.searchParams.set("resource", "acct:someone@localhost:8000");
+    const context = createContext(u);
+    const request = context.request;
+    const response = await handleWebFinger(request, {
+      context,
+      actorDispatcher,
+      onNotFound,
+    });
+    assertEquals(response.status, 200);
+    assertEquals(response.headers.get("Content-Type"), "application/jrd+json");
+    assertEquals(response.headers.get("Access-Control-Allow-Origin"), "*");
+    assertEquals(await response.json(), expectedForLocalhostWithPort);
+  });
+
+  const expectedForHostnameWithPort = {
+    subject: "acct:someone@example.com:8000",
+    aliases: ["http://example.com:8000/users/someone"],
+    links: [
+      {
+        href: "http://example.com:8000/users/someone",
+        rel: "self",
+        type: "application/activity+json",
+      },
+      {
+        href: "http://example.com:8000/@someone",
+        rel: "http://webfinger.net/rel/profile-page",
+      },
+      {
+        href: "https://example.org/@someone",
```

Comment:
I think you missed one?

```suggestion
        href: "https://example.org:8000/@someone",
```

Reply:
@ThisIsMissEm I left it intentionally assuming that it's alternate url and not based on `actorUri`
for the other testcase "localhost with port", I left it as "example.org". 

https://github.com/fedify-dev/fedify/blob/d3ac8f8734fb6fe522a95e8830ebf40526ad9be2/src/webfinger/handler.test.ts#L128-L136

Do I need to make change in both places?